x-pack/test/functional/services/transform/edit_flyout.ts

```diff
@@ -52,6 +52,28 @@ export function TransformEditFlyoutProvider({ getService }: FtrProviderContext)
       );
     },
 
+    async assertTransformEditFlyoutRetentionPolicyFieldSelectValue(expectedValue: string) {
+      await testSubjects.existOrFail(`transformEditFlyoutRetentionPolicyFieldSelect`, {
+        timeout: 1000,
+      });
+      const actualValue = await testSubjects.getAttribute(
+        'transformEditFlyoutRetentionPolicyFieldSelect',
+        'value'
+      );
+      expect(actualValue).to.eql(
+        expectedValue,
+        `Detector input option value should be '${expectedValue}' (got '${actualValue}')`
```

Comment:
We should adjust the failure message to talk about retention policy field.

Reply:
Oh yes good catch on that copy/paste error! Fixed in 61ed8e4.